pkg/data/task_result.go

```diff
@@ -27,6 +27,7 @@ type TaskOutput struct {
 	Import    *ImportTaskOutput    `json:"import,omitempty"`
 	Export    *ExportTaskOutput    `json:"export,omitempty"`
 	Transcode *TranscodeTaskOutput `json:"transcode,omitempty"`
+	Prepare   *PrepareTaskOutput   `json:"prepare,omitempty"`
```

Comment:
The keys here should correspond to the task types that the API supports. So I'm not so sure if we should have this here, unless there is any explicit "prepare task" created by the API sometimes.

That might be the case for example for the old assets which we haven't prepared yet, or maybe for when the import/transcode tasks work but the prepare fails and we still want to retry the prepare logic. Hmmm, ok, if it is for one of those cases I think it makes sense to keep this here. Otherwise, or if we we will only add those use cases in the future, maybe we're ok with only the `PlaybackRecordingSessionId` field in the `AssetSpec`.

Reply:
Yes I am thinking to support `prepare` as a standalone task in the future in case of failure/retry. It's not currently implemented as it's own task though this is just some pre-wiring, so I can get rid of it until later, just use `PlaybackRecordingSessionId` in `AssetSpec` for now.